Libraries/CustomComponents/Navigator/NavigatorNavigationBar.js

```diff
@@ -145,6 +145,7 @@ var NavigatorNavigationBar = React.createClass({
       var component = this._components[componentName].get(this.props.navState.routeStack[index]);
       var props = this._getReusableProps(componentName, index);
       if (component && interpolate[componentName](props.style, amount)) {
+        props.pointerEvents = props.style.opacity === 0 ? 'none' : 'box-none';
```

Comment:
This is a cheap check. The main thing we want to avoid is actually setting the native property every frame if we can help it, which in this case we should, since most of the time we're not changing the value of the pointerEvents property.